base/system/services/rpcserver.c

```diff
@@ -116,7 +116,7 @@ ScmStartRpcServer(VOID)
                                     NULL);
     if (Status != RPC_S_OK)
     {
-        DPRINT1("RpcServerUseProtseqEpW() failed (Status %lx)\n", Status);
+        DPRINT("RpcServerUseProtseqEpW() failed (Status %lx)\n", Status);
```

Comment:
Please restore all these unrelated (DPRINT1 -> DPRINT) changes.

Reply:
My gutfeeling is the same as that of Hermes. Those DPRINT1 which are only triggered in error-cases should most likely be kept to be a DPRINT1. But I will not make my approval dependent on that. I gave it already.

Reply:
Indeed this was the purpose of having DPRINT1's only in the failure paths.

Reply:
DPRINT1's now used and only used on failure paths. Thanks.